modules/builds-running-entitled-builds-with-sharedsecret-objects.adoc

```diff
@@ -29,169 +22,103 @@ If neither of the last two prerequisites in this list are met, establish, or ask
 
 .Procedure
 
-. Grant the `builder` service account RBAC permissions to use the `SharedSecret` CR instance by using `oc apply` with YAML content:
+. Use `oc apply` to create a `SharedSecret` object instance with the cluster's entitlement secret.
 +
-[NOTE]
+[IMPORTANT]
 ====
-Currently, `kubectl` and `oc` have hard-coded special case logic restricting the `use` verb to roles centered around pod security. Therefore, you cannot use `oc create role ...` to create the role needed for consuming `SharedSecret` CR instances.
+You must have cluster administrator permissions to create `SharedSecret` objects.
 ====
 +
 .Example `oc apply -f` command with YAML `Role` object definition
 [source,terminal]
 ----
 $ oc apply -f - <<EOF
+kind: SharedSecret
+apiVersion: sharedresource.openshift.io/v1alpha1
+metadata:
+  name: etc-pki-entitlement
+spec:
+  secretRef:
+    name: etc-pki-entitlement
+    namespace: openshift-config-managed
+EOF
+----
+
+. Create a role to grant the `builder` service account permission to access the `SharedSecret` object by using the following YAML content:
++
+.Example `oc apply -f` command
+[source,terminal]
+----
+$ oc apply -f - <<EOF
 apiVersion: rbac.authorization.k8s.io/v1
 kind: Role
 metadata:
-  name: shared-resource-my-share
-  namespace: my-namespace
+  name: builder-etc-pki-entitlement
 rules:
   - apiGroups:
       - sharedresource.openshift.io
     resources:
       - sharedsecrets
     resourceNames:
-      - my-share
+      - etc-pki-entitlement
     verbs:
       - use
 EOF
 ----
 
-. Create the `RoleBinding` associated with the role by using the `oc` command:
+. Create a `RoleBinding` object that grants the `builder` service account permission to access the `SharedSecret` object by running the following command:
 +
 .Example `oc create rolebinding` command
 [source,terminal]
 ----
-$ oc create rolebinding shared-resource-my-share --role=shared-resource-my-share --serviceaccount=my-namespace:builder
+$ oc create rolebinding builder-etc-pki-entitlement --role=builder-etc-pki-entitlement --serviceaccount=builder
```

Comment:
Checking the CLI docs - we must have a namespace for the service account.

Let's use `build-namespace` as the placeholder.